metabase/metabase
# Pie chart labels

Epic: https://github.com/metabase/metabase/issues/24877

## Changes

Support labels on pie charts:
- Added "Show data labels" setting
- Labels hide when there is not enough space

## How to verify
- Create a pie chart
- Enable "Show data labels" setting
- Ensure labels are visible if pie arcs can fit them in

## Screenshots

Big
<img width="1001" alt="Screen Shot 2022-08-30 at 1 05 16 AM" src="https://user-images.githubusercontent.com/14301985/187300525-72d52988-aa42-4098-bdb2-b24136c28f74.png">

Medium
<img width="695" alt="Screen Shot 2022-08-30 at 1 05 33 AM" src="https://user-images.githubusercontent.com/14301985/187300581-7609518b-c1a1-4289-836e-85e193d981e2.png">

Small
<img width="613" alt="Screen Shot 2022-08-30 at 1 05 41 AM" src="https://user-images.githubusercontent.com/14301985/187300608-40f7199d-bdd8-4748-bf59-d0dba3ad677a.png">
